Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -177,9 +182,10 @@ def http_request(
         # 206 indicates Partial Content, reason will be in the warning header.
         # In that case, logs with the warning header will be written.
         if response.status_code == 206:
-            demisto.debug(str(response.headers))
-        is_response_empty_and_successful = (response.status_code == 204)
-        if is_response_empty_and_successful and return_empty_response:
+            demisto.debug(f'Response status code 206 (partial content) with headers {str(response.headers)}')
+
+        allowed_empty_response_codes = kwargs.get('empty_valid_codes', (204,))  # 204 is CommonServerPython's default
```

Comment:
I don't understand, what the meaning of `CommonServerPython's default`?

Reply:
https://github.com/demisto/content/blob/master/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L8531-L8533